beamer/tests/check/test_invalidations.py

```diff
@@ -0,0 +1,58 @@
+import json
+from pathlib import Path
+
+import ape
+import apischema
+import pytest
+
+import beamer.check.commands
+from beamer.check.commands import Invalidation
+
+from beamer.tests.util import deploy, get_repo_root, run_command, write_keystore_file
+
+
+@pytest.mark.parametrize("count", (1, 3))
+def test_initiate_l1_invalidations(tmp_path, deployer, count):
+    password = "test"
+    keystore_file = tmp_path / f"{deployer.address}.json"
+    write_keystore_file(keystore_file, deployer.private_key, password)
+
+    rpc_file, artifact = deploy(deployer, tmp_path)
+
+    root = get_repo_root()
+    output = tmp_path / "invalidations.json"
+    chain_id = ape.chain.chain_id
+    run_command(
+        beamer.check.commands.initiate_l1_invalidations,
+        "--keystore-file",
+        keystore_file,
+        "--password",
+        password,
+        "--rpc-file",
+        rpc_file,
+        "--abi-dir",
+        f"{root}/contracts/.build/",
+        "--artifacts-dir",
+        Path(artifact).parent,
+        "--output",
+        output,
+        "--count",
+        count,
+        str(chain_id),
+        str(chain_id),
```

Comment:
Since you test the count variable here, you could also test that it works with multiple proof targets (I think the existence of the chain is not checked on a contract level during invalidation).

Reply:
Ah, good point; added a new, dummy proof target.